package.json

```diff
@@ -22,7 +22,8 @@
     "start:dev-server": "nodemon server --watch server --watch build --watch content --watch kumascript",
     "start:static-server": "cross-env ENV_FILE=testing/.env node server/static.js",
     "filecheck": "cd filecheck && node cli.js",
-    "redirects": "node redirects/cli.js"
+    "redirects": "node tool/cli.js redirect",
```

Comment:
If you're supposed to type `yarn tool content-move ... ...` now, why bother keeping this line? Seems redundant these days. 

What matters is what we do we put into the `scripts` on mdn/content which is the "formal CLI" for writers.